Modules/Module_Import/xASL_wrp_NII2BIDS_Subject.m

```diff
@@ -90,7 +88,7 @@
 					checkExpression = regexp(nameSubjectSession, [separator visitAliases{iAlias,1} '$'], 'once');
 				end
 				if ~isempty(checkExpression) % nameSubject should end in the visit alias
-					visitName = nameSubjectSession(checkExpression:end);
+					visitName = nameSubjectSession(checkExpression+1:end);
```

Comment:
Are you sure this doesn't break old code?

Reply:
Yes. VERY sure. This is Michael nonsense of saving visit names as "_1" instead of "1". 

So I cleaned all code and save really only the visit name and always add the underscore when necessary to connect into a filename. But having an underscore in the visit name was sooo dirty...

Now you see why I complained so much :)